docs/content/policy-reference.md

```diff
@@ -604,9 +604,53 @@ The table below shows examples of calling `http.send`:
 | Built-in | Description |
 | ------- |-------------|
 | <span class="opa-keep-it-together">``net.cidr_contains(cidr, cidr_or_ip)``</span> | `output` is `true` if `cidr_or_ip` (e.g. `127.0.0.64/26` or `127.0.0.1`) is contained within `cidr` (e.g. `127.0.0.1/24`) and false otherwise. Supports both IPv4 and IPv6 notations.|
+| <span class="opa-keep-it-together">``output := net.cidr_contains_matches(cidrs, cidrs_or_ips)``</span> | `output` is a `set` of tuples identifying matches where `cidrs_or_ips` are contained within `cidrs`. This function is similar to `net.cidr_contains` except it allows callers to pass collections of CIDRs or IPs as arguments and returns the matches (as opposed to a boolean result indicating a match between two CIDRs/IPs.) See below for examples. |
 | <span class="opa-keep-it-together">``net.cidr_intersects(cidr1, cidr2)``</span> | `output` is `true` if `cidr1` (e.g. `192.168.0.0/16`) overlaps with `cidr2` (e.g. `192.168.1.0/24`) and false otherwise. Supports both IPv4 and IPv6 notations.|
 | <span class="opa-keep-it-together">``net.cidr_expand(cidr)``</span> | `output` is the set of hosts in `cidr`  (e.g., `net.cidr_expand("192.168.0.0/30")` generates 4 hosts: `{"192.168.0.0", "192.168.0.1", "192.168.0.2", "192.168.0.3"}` |
 
+#### `net.cidr_contains_matches` examples
+
+The `output := net.cidr_contains_matches(a, b)` function allows callers to supply
+strings, arrays, sets, or objects for either `a` or `b`. The `output` value in
+all cases is a set of tuples (2-element arrays) that identify matches, i.e.,
+elements of `b` contained by elements of `a`. The first tuple element refers to
+the match in `a` and the second tuple element refers to the match in `b`.
+
+| Input Type | Output Type |
+| --- | --- |
+| `string` | `string` |
+| `array` | `array` index |
+| `set` | `set` element |
+| `object` | `object` key |
+
+
+```live:netcidrcontainsmatches/strings:query:merge_down
```

Comment:
Might need to add in a like

\`\`\`live:netcidrcontainsmatches:module:hidden
package example
\`\`\`

To make the live block build happy